test/end2end/app_actions.dart

```diff
@@ -322,11 +322,21 @@ class AppActions {
     await tester.pumpAndSettle();
 
     // Check that the post content is displayed
-    expect(find.text(comment), findsOneWidget);
+    final commentText = find.text(comment);
+    expect(commentText, findsOneWidget);
 
-    // Find the delete button on card
-    final deleteButton = find.byKey(ProfileInfoCard.deleteButtonCardKey);
-    expect(deleteButton, findsOneWidget);
+    // Find and tap the delete button on the card
+    final commentCard = find.ancestor(
+      of: find.text(comment),
```

Comment:
```suggestion
      of: commentText,
```

We can reuse the finder we declared earlier for more clarity.

Reply:
I agree, this is cleaner this way! :)